### DISCUSSION TITLE
[6 giugno 2024] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#28] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/bfb5c551-8ff2-4331-a34b-b9a00e69703f)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo le seguenti due soluzioni presenti nei miei appunti:

![image](https://github.com/user-attachments/assets/b253faee-4fcf-4561-8cfd-917801af815a)
![image](https://github.com/user-attachments/assets/2482b3e0-7f7f-4b1b-9b08-07ad838f3d33)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

L'esempio di applicazione coincide con quello dell'esercizio #7

**Comment by daromacarlo** [comment]
<img width="1086" height="588" alt="immagine" src="https://github.com/user-attachments/assets/acfff87b-bef3-4277-a034-5618f76eb4d5" />
<img width="995" height="802" alt="immagine" src="https://github.com/user-attachments/assets/81fe3129-1364-4882-bb3b-a5b4117666d9" />
